### ISSUE TITLE
Integrate RDR tagger

### ISSUE THREAD

**Issue by zesch** [dkpro/dkpro-core#820]
We should check whether we want to integrate this tagger
http://rdrpostagger.sourceforge.net

**Comment by reckart** [comment]
Just added that to my queue today :) The questions would IMHO rather be:

* why not add it?
* who volunteers?

**Comment by zesch** [comment]
@Horsmann Interested in adding a new tagger? :)

**Comment by reckart** [comment]
@zesch for curiosity: what makes this tagger particularly interesting to you?

**Comment by zesch** [comment]
Morphological tags for a lot of languages sound promising, but unsure what kind of performance is to be expected.

**Comment by reckart** [comment]
There seem to be a few quite exotic languages indeed :)

**Comment by daxenberger** [comment]
FYI: one of the developers has been doing an internship at UKP Lab a while ago ;-)

**Comment by reckart** [comment]
Really? I did check the publication history and it appears to be another person by the same name to me.

**Comment by daxenberger** [comment]
> Really?

yes. See the bio.

**Comment by Horsmann** [comment]
@reckart how would you integrate a java application that is only released to a software repository as code but is not available as maven release. In this case, the developer uses a versioning schema but this is not reflected in the git-log so far I see. The `latest` version the developer distinguishes is from May, the last code changes are from September. How to integrate this properly into Core?

**Comment by reckart** [comment]
A third-party deploy to Maven central would be necessary.

banana-split (a compound splitter) was at the time also only released as a ZIP file on a website (I don't know it's status now). What I did was:

* ask the developers if they agree that I do a third-party upload to Maven Central (and propose a groupId/artifactId) or if they want to do it themselves: they said I could do it
* download the ZIP
* create a local Maven project using the source code from the ZIP
* asked Sonatype what to put as the SCM URL into the pom - we agreed to simply put the URL of the ZIP file
* build the JAR and source JAR from the local project
* uploaded them to Maven Central as a third-party upload

Then I used the resulting artifact in DKPro Core.
